src/components/MyFeed/Carousel/index.js

```diff
@@ -0,0 +1,297 @@
+    zIndex: "-1",
+    position: "relative",
+    right: "60%",
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "space-evenly",
+    filter: "brightness(0.74) blur(2px)",
+  },
+  media: {
+    height: 140,
+  },
+  arrow: {
+    "&:hover": {
+      transform: "scale(1.2,1.2)",
+    },
+  },
+  "@keyframes myEffect": {
+    "0%": {
+      opacity: 1,
+
+      transform: "scale(.6,.6) ",
+    },
+    "100%": {
+      opacity: 1,
+      transform: "scale(1,1) rotateY(0)",
+    },
+  },
+  "@keyframes myEffectRoot": {
+    "0%": {
+      opacity: 1,
+
+      transform: "scale(.6,.6) rotateY(-100deg)",
+    },
+    "100%": {
+      opacity: 1,
+      transform: "scale(1,1) rotateY(0)",
+    },
+  },
+}));
+
+const Carousel = () => {
+  const classes = useStyles();
+  const [current, setCurrent] = useState(0);
+  const launchedOrgs = useSelector(({ org }) => org.launched.data);
+  const dispatch = useDispatch();
+  const firestore = useFirestore();
+  const [width, setWidth] = useState(window.innerWidth);
+  useEffect(() => {
+    const handleResize = () => {
+      setWidth(window.innerWidth);
+    };
+    window.addEventListener("resize", handleResize);
+  });
+  var length = 0;
+  length = launchedOrgs && launchedOrgs.length;
+
+  useEffect(() => {
+    getLaunchedOrgsData()(firestore, dispatch);
+    return () => {
+      clearOrgData()(dispatch);
+    };
+  }, [firestore, dispatch]);
+
+  const nextSlide = () => {
+    setCurrent(current === length - 1 ? 0 : current + 1);
+  };
+
+  const prevSlide = () => {
+    setCurrent(current === 0 ? length - 1 : current - 1);
+  };
+
+  return (
+    <section className={classes.slides}>
+      <ArrowBackIosIcon
+        onClick={prevSlide}
+        className={classes.arrow}
+        style={{ position: "absolute", left: "-1rem" }}
+      />
+      <ChevronRightIcon
+        onClick={nextSlide}
+        className={classes.arrow}
+        style={{ position: "absolute", right: "-1rem", fontSize: "2.3rem" }}
+      />
+
+      {width > 800
+        ? launchedOrgs &&
+          launchedOrgs.map((org, index) => {
+            return (
+              <Grid
+                item
+                className={
+                  index === current ? `active ${classes.slide}` : classes.slide
+                }
+                key={index}
+              >
+                {index === (current - 1 >= 0 ? current - 1 : length - 1) && (
+                  <Link to={`/org/${org.org_handle}`}>
+                    <Card
+                      className={classes.rootLeft}
+                      style={{
+                        zIndex: -1,
+                      }}
+                    >
+                      <CardActionArea>
+                        <CardMedia
+                          className={classes.media}
+                          alt="CodeLabz"
+                          component="img"
+                          title="CodeLabz"
+                          // image={org.org_image}
```

Comment:
If no longer needed, remove these kind of comments

Reply:
ok i am removing unnecessary code

Reply:
fixed